src/main/java/com/google/opensesame/user/UserData.java

```diff
@@ -139,4 +139,17 @@ private void loadProjects(ArrayList<String> projectIds) throws IOException {
       projects.add(curProject);
     }
   }
+
+  /**
+   * @param {UserData} user2 the user to compare to
+   * @return an integer 'compatibility score' which represents how many shared interests two users
+   *     have.
+   */
+  public int compatibility(UserData user2) {
+    ArrayList<String> user1Interests = (ArrayList) this.interestTags.clone();
+    ArrayList<String> user2Interests = (ArrayList) user2.interestTags.clone();
+    user1Interests.retainAll(user2Interests);
+    System.out.println(user1Interests.size());
```

Comment:
Is this intended to be here? If so, it might be worth adding a TODO to remove this for production.

Reply:
If we're going to keep this line in print some context along with the int so that we know what it is. If we just have print statements that print numbers or strings without additional context it makes the logs unreadable when 10+ print statements from different functions get called.

Reply:
So sorry, I definitely meant to delete this! Joe, thank you for the advice for future print statements.